gke-deploy/README.md

```diff
@@ -14,35 +14,38 @@ the following:
 
 ### Prepare Step
 
-1.  `gke-deploy` modifies a set of Kubernetes resource YAML configs to use a
-    container image's digest instead of a tag and adds
-    [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#applications-and-instances-of-applications).
+* The `gke-deploy` builder modifies a set of Kubernetes resource YAML configs to use a
+container image's digest instead of a tag.
+* This builder adds [recommended
+labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#applications-and-instances-of-applications).
 
 ### Apply Step
 
-1.  `gke-deploy` gets authorized to access a GKE cluster.
+The `gke-deploy`builder does the following:
 
-2.  `gke-deploy` applies the set of Kubernetes resource YAML configs that were
-    modified in the prepare step.
+1. Gets authorized to access a GKE cluster
 
-3.  `gke-deploy` waits for applied Kubernetes resources to be ready.
+2. Applies the set of Kubernetes resource YAML configs that were
+modified in the prepare step
+
+3. Waits for applied Kubernetes resources to be ready
 
 ## Usage
 
-### [gke-deploy run [flags]](docs/run.md)
+`[gke-deploy run [flags]](docs/run.md)`
 
 This command will execute both the [Prepare](#prepare-step) and
 [Apply](#apply-step) phases mentioned above.
 
-### [gke-deploy prepare [flags]](docs/prepare.md)
+`[gke-deploy prepare [flags]](docs/prepare.md)`
 
 This command will execute only the [Prepare](#prepare-step) phase mentioned
 above.
 
-### [gke-deploy apply [flags]](docs/apply.md)
+`[gke-deploy apply [flags]](docs/apply.md)`
 
 This command will execute only the [Apply](#apply-step) phase mentioned above.
 
 ### [Automated Deployments with GCB](docs/automated-deployments.md)
```

Comment:
What do you think about changing ### to ## and moving this out of the Usage section?